packages/runtime-core/src/apiCreateApp.ts

```diff
@@ -103,12 +103,16 @@ export function createAppAPI<HostNode, HostElement>(
       use(plugin: Plugin, ...options: any[]) {
         if (installedPlugins.has(plugin)) {
           __DEV__ && warn(`Plugin has already been applied to target app.`)
+        } else if (
+          plugin &&
+          'install' in plugin &&
```

Comment:
This line seems unnecessary.

Reply:
Typescript complains without it (`plugin` still includes `PluginInstallFunction`), it was either this or a bunch of type assertion.